stdlib/universe/filter_test.go

```diff
@@ -580,6 +582,143 @@ func TestFilterOperation_Marshaling(t *testing.T) {
 	querytest.OperationMarshalingTestHelper(t, data, op)
 }
 
+func TestMergeFilterAnyRule(t *testing.T) {
+	var (
+		from        = &influxdb.FromProcedureSpec{}
+		count       = &universe.CountProcedureSpec{}
+		filterOther = &universe.FilterProcedureSpec{
+			Fn: &semantic.FunctionExpression{
+				Block: &semantic.FunctionBlock{
+					Body: &semantic.IdentifierExpression{
+						Name: "foo",
+					},
+				},
+			},
+		}
+		filterTrue = &universe.FilterProcedureSpec{
+			Fn: &semantic.FunctionExpression{
+				Block: &semantic.FunctionBlock{
+					Body: &semantic.BooleanLiteral{
+						Value: true,
+					},
+				},
+			},
+		}
+		filterFalse = &universe.FilterProcedureSpec{
+			Fn: &semantic.FunctionExpression{
+				Block: &semantic.FunctionBlock{
+					Body: &semantic.BooleanLiteral{
+						Value: false,
+					},
+				},
+			},
+		}
+	)
+
+	tests := []plantest.RuleTestCase{
+		{
+			Name: "filterOther",
+			// from -> filter => from -> filter
+			Rules: []plan.Rule{universe.MergeFilterAnyRule{}},
+			Before: &plantest.PlanSpec{
+				Nodes: []plan.PlanNode{
+					plan.CreatePhysicalNode("from", from),
+					plan.CreatePhysicalNode("filter", filterOther),
+				},
+				Edges: [][2]int{{0, 1}},
+			},
+			After: &plantest.PlanSpec{
```

Comment:
These tests can be made a bit more concise by using `RuleTestCase.NoChange` to indicate that no rewrite should be done.

Reply:
great